CineElorrietaMVC/src/main/java/Controlador/ControladorPanelPeliculas.java

```diff
@@ -0,0 +1,83 @@
+package Controlador;
+
+import java.util.ArrayList;
+
+import javax.swing.DefaultListModel;
+import javax.swing.JList;
+import javax.swing.JOptionPane;
+
+import Modelo.Modelo;
+import Modelo.Pelicula;
+import Vista.PanelPeliculas;
+import Vista.Vista;
+import Vista.PanelGeneros;
+
+
+
+public class ControladorPanelPeliculas {
+
+	@SuppressWarnings("unused")
+	private Modelo modelo;
+	private Vista vista;
+	private Controlador controlador;
+	private PanelPeliculas panelPeliculas;
+	
+	
+	public ControladorPanelPeliculas(Modelo modelo, Vista vista, Controlador controlador) {
+		this.modelo = modelo;
+		this.vista = vista;
+		this.controlador = controlador;	
+	}
+	
+	public void mostrarPanelPeliculas(String genero) {
+		
+		this.panelPeliculas = new PanelPeliculas(this,genero);
+		this.vista.mostrarPanel(this.panelPeliculas);
+
+	
+		
+	}
+	
+	public void accionadoBotonVolverPanelPeliculas() {
+		controlador.navegarPanelGeneros();
+		
+	}
+
+	
+	public void accionadoBotonAnadirPanelPeliculas(String peliSeleccionada) {
+		modelo.introducirPeliculaSeleccionada(peliSeleccionada,controlador,controlador.getModelo().getPeliculas());
```

Comment:
Necesitas el controlador en el modelo? Lo digo porque ya le estás pasando las películas. Aunque, lo que me sorprende aun más es que le estás pasando las películas del modelo al mismo modelo. El modelo no tiene ya esta información? Estáis usando más de un modelo?

Reply:
Cambio realizado